src/PrestaShopBundle/Form/Admin/Improve/Shipping/Carrier/Type/CostsRangeType.php

```diff
@@ -46,13 +47,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('range', TextPreviewType::class, [
                 'label' => $this->trans('Range', 'Admin.Shipping.Feature'),
             ])
-            ->add('from', HiddenType::class)
-            ->add('to', HiddenType::class)
+            ->add('from', HiddenType::class, [
+                'constrains' => [
+                    new PositiveOrZero([
+                        'message' => 'The value must be a positive number.',
+                    ]),
+                ],
+            ])
+            ->add('to', HiddenType::class, [
+                'constrains' => [
+                    new PositiveOrZero([
```

Comment:
I wonder if this one should be strictly positive, but that's just a suggestion/question I'm asking myself I guess QA or user feedback will let us know